atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172111 implement product payment

#### What does this PR do?
-ensure payment is done when the card is valid
-ensure the buyer is redirected to confirmation page in case the card requires additional steps -ensure the payment is declined if the card has insufficient amount of money to pay

#### Description of Task to be completed?
-payment of an order

#### How should this be manually tested?
- first you need to login
- go to the payment page using /payment?rid='order-id'
- use this cards to simulate payment scenarios 

    1. 4242424242424242  ( successful payment)
    2. 4000000000009995  ( failed payment )
    3. 4000002500003155   ( payment that require additional authentication)

- after clicking the pay button you should get the response/ redirected to confirmation page based on the card you used

#### Screenshots (if appropriate)
payment form

<img width="960" alt="updated" src="https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/99591226/6dd77030-f848-499c-ad11-b31f4b81eebd">

confirmation page
<img width="632" alt="confirm" src="https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/99591226/f7f39ba0-8dac-43ec-83de-4e538b64fb23">

#### What are the relevant pivotal tracker/Trello stories?
#185172111
